operations/helm/charts/mimir-distributed/templates/alertmanager/alertmanager-statefulset.yaml

```
@@ -16,6 +16,11 @@ metadata:
   namespace: {{ .Release.Namespace | quote }}
 spec:
   replicas: {{ $rolloutZone.replicas }}
+  {{- if and (semverCompare ">= 1.23-0" (include "mimir.kubeVersion" .)) (.Values.alertmanager.persistentVolume.enableStatefulSetAutoDeletePVC)  }}
```

sorry for the back and forth but I would expect that if `enableStatefulSetAutoDeletePVC` is eabled, then my PVCs are autodeleted. In this case this only means that the StatefulSet gets the `persistentVolumeClaimRetentionPolicy`. I think we should rename this to `.Values.alertmanager.persistentVolume.enableRetentionPolicy`
```suggestion
  {{- if and (semverCompare ">= 1.23-0" (include "mimir.kubeVersion" .)) (.Values.alertmanager.persistentVolume.enableRetentionPolicy)  }}
```